server/revision/revision.service.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+const { Revision, Sequelize } = require('../shared/database');
+
+const { Op } = Sequelize;
+
+function getByEntity(query, options) {
```

Comment:
`repositoryId` is inherited code, but since we're uniquely filtering using JSON, we might leverage it to speed things up. I didn't look further into this. What do you think?